priv/repo/migrations/20170809102200_emails_location_for_prod.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Askbywho.Repo.Migrations.EmailsLocationForProd do
+  use Ecto.Migration
+
+  def change do
+    alter table(:emails) do
+      add :latitude, :real
+      add :longitude, :real
+      add :location, :string
```

Comment:
ok, but its assuming your geoip is current and getting the right lat/long, just seems like having the original could be useful.  Not a huge deal